nss-day-cohort-43/nuttyshell-the-coven-code
# Se add friend post section

# Description
Ability to add friend by clicking on usernames in post section.
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Try adding:
- a friend, deleting friend, re-adding
- activeUser
- already added friends
See how it works with switching out users
# Checklist:
